$NetBSD$

* Relax dependency: allow newer Rails 8.1, rubyzip, commonmarker,
  net-ldap, mini_magick and sqlite3 releases.
* Do not load gem for development and test.

--- Gemfile.orig
+++ Gemfile
@@ -2,7 +2,7 @@
 
 ruby '>= 3.2.0', '< 4.1.0'
 
-gem 'rails', '8.1.4'
+gem 'rails', '~> 8.1.4'
 gem 'rouge', '~> 5.0'
 gem 'mini_mime', '~> 1.1.0'
 gem "actionpack-xml_parser"
@@ -13,12 +13,12 @@
 gem 'i18n', '~> 1.15.2'
 gem 'rbpdf', '~> 1.21.4'
 gem 'addressable'
-gem 'rubyzip', '~> 3.4.0'
+gem 'rubyzip', '>= 3.4.0'
 gem 'propshaft', '~> 1.3.0'
 gem 'rack', '>= 3.1.3'
 gem "stimulus-rails", "~> 1.3"
 gem "importmap-rails", "~> 2.0"
-gem 'commonmarker', '~> 2.8.2'
+gem 'commonmarker', '>= 2.8.2'
 gem "doorkeeper", "~> 5.8.2"
 gem "bcrypt", require: false
 gem "doorkeeper-i18n", "~> 5.2"
@@ -43,12 +43,12 @@
 
 # Optional gem for LDAP authentication
 group :ldap do
-  gem 'net-ldap', '~> 0.20.0'
+  gem 'net-ldap', '>= 0.20.0'
 end
 
 # Optional gem for exporting the gantt to a PNG file
 group :minimagick do
-  gem 'mini_magick', '~> 5.2.0'
+  gem 'mini_magick', '>= 5.2.0'
 end
 
 # Include database gems for the adapters found in the database
@@ -79,7 +79,7 @@
       when /postgresql/
         gem 'pg', '~> 1.6.2'
       when /sqlite3/
-        gem 'sqlite3', '~> 2.9.4'
+        gem 'sqlite3', '>= 2.9.4'
       when /sqlserver/
         gem 'tiny_tds', '~> 3.4'
         gem 'activerecord-sqlserver-adapter', '~> 8.1.2'
@@ -94,6 +94,7 @@
   warn("Please configure your config/database.yml first")
 end
 
+if false
 group :development, :test do
   gem 'debug'
 end
@@ -123,6 +124,7 @@
   gem 'oauth2'
   gem 'webrick'
 end
+end
 
 local_gemfile = File.join(File.dirname(__FILE__), "Gemfile.local")
 if File.exist?(local_gemfile)
